yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -112,6 +113,11 @@ func assessMigrationBulk() {
 				break
 			}
 		}
+		if ProcessShutdownRequested {
+			log.Info("Shutting down as SIGINT/SIGTERM received...")
+			log.Infof("sleep for 10 seconds for exit handlers to execute")
+			time.Sleep(time.Second * 10)
```

Comment:
Here, after 10s if exit handlers did not successfully complete, we will continue with the next assessment; This should not happen. Let's just return from the function here after sleeping

Reply:
ok

Reply:
When cleaning child processes, we have a timeout of 60s (`err := ShutdownProcess(pid, 60)`); Should we wait for at least 60 seconds here?

Reply:
60 seconds will be too much here.
In `ShutdownProcess` it is like max 60sec, voyager can exit before

Reply:
as discussed, better approach will be just return and the main thread waiting for the cleanup goroutine to exit.

Will be completely fixed under this ticket - https://yugabyte.atlassian.net/issues/DB-12388